docs/advanced-features/s3-prefix-permissions.md

```diff
@@ -1,13 +1,18 @@
-# Restricting access to specific prefixes in a S3 bucket
+<!-- markdownlint-disable -->
```

Comment:
```suggestion
```

I disabled violated rule in https://github.com/quiltdata/quilt/pull/3302.
For future, it's possible to disable specific rules instead of disabling everything: https://github.com/DavidAnson/markdownlint#configuration.